dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task221018CreateVariantFieldInMultiTreeTest.java

```diff
@@ -94,16 +111,18 @@ private static void cleanUp () throws SQLException {
                 .orElse("multi_tree_pkey");
 
         try {
-            new DotConnect().executeStatement(
-                    "ALTER TABLE multi_tree DROP COLUMN variant_id");
-        } catch (SQLException e) {
+            DotDatabaseMetaData.dropPrimaryKey("multi_tree");
+        } catch (Exception e) {
             //ignore
+            System.out.println();
```

Comment:
can this be removed?

Reply:
done https://github.com/dotCMS/core/pull/23270/commits/f2da06cdf6fa6d1041394dc24d1d0f0b03fa972f